contracts/FUSD.cdc

```diff
@@ -150,58 +194,37 @@ pub contract FUSD: FungibleToken {
         init() {
             self.minterCapability = nil
         }
-
-    }
-
-    // createMinterProxy
-    //
-    // Function that creates a MinterProxy.
-    // Anyone can call this, but the MinterProxy cannot mint without a Minter capability,
-    // and only the admin can provide that.
-    //
-    pub fun createMinterProxy(): @MinterProxy {
-        return <- create MinterProxy()
     }
 
     // Administrator
-    //
-    // A resource that allows new minters to be created
-    //
-    // We will only want one minter for now, but might need to add or replace them in future.
-    // The Minter/Minter Proxy structure enables this.
-    // Ideally we would create this structure in a single function, generate the paths from the address
-    // and cache all of this information to enable easy revocation but String/Path comversion isn't yet supported.
-    //
-    pub resource Administrator {
-
-        // createNewMinter
-        //
-        // Function that creates a Minter resource.
-        // This should be stored at a unique path in storage then a capability to it wrapped
-        // in a MinterProxy to be stored in a minter account's storage.
-        // This is done by the minter account running:
-        // transactions/fusd/minter/setup_fusd_minter.cdc
-        // then the admin account running:
-        // transactions/fusd/admin/deposit_fusd_minter.cdc
-        //
-        pub fun createNewMinter(): @Minter {
-            emit MinterCreated()
-            return <- create Minter()
-        }
-
-    }
+    // kept for backwards compatibility
+    access(all) resource Administrator {}
 
     init() {
         self.AdminStoragePath = /storage/fusdAdmin
         self.MinterProxyPublicPath = /public/fusdMinterProxy
         self.MinterProxyStoragePath = /storage/fusdMinterProxy
 
-        self.totalSupply = 0.0
+        self.totalSupply = 1000.0
```

Comment:
Confused by the new code in init, does init run again on 1.0 migration? Or is this just for previewnet?

Reply:
just for testing, and yeah I guess it will also run on previewnet